### PR TITLE
Bug 1515798 - Convert login.jsx into a reactstrap component

### PR DESCRIPTION
**Pull request by KWierso**

### CODE REVIEW

File: ui/shared/auth/Login.jsx [hunk 1]
```diff
@@ -105,45 +112,40 @@ class Login extends React.Component {
...
         {!user.isLoggedIn && (
-          <span className="btn nav-login-btn" onClick={this.login}>
-            {' '}
-            Login / Register
-          </span>
+          <Button
+            className="btn btn-view-nav nav-login-btn"
+            onClick={this.login}
+          >
+            <span className="fa fa-user-plus" /> Login / Register
```
Comment (camd) [inline ui/shared/auth/Login.jsx]: I think the ``fa-sign-in`` icon is more appropriate for this.

File: ui/shared/auth/Login.jsx [hunk 2]
```diff
@@ -105,45 +112,40 @@ class Login extends React.Component {
...
-              </li>
-            </ul>
-          </span>
+              <div className="nav-user-name">
+                <span>{user.fullName}</span>
+              </div>
+            </DropdownToggle>
+            <DropdownMenu aria-labelledby="logoutLabel">
+              <DropdownItem onClick={this.logout}>
+                <span className="dropdown-item">
```
Comment (camd) [inline ui/shared/auth/Login.jsx]: this span with the ``dropdown-item`` class is not necessary anymore.  I think you can just remove it and leave the inner ``span``

File: ui/shared/auth/Login.jsx [hunk 3]
```diff
@@ -105,45 +112,40 @@ class Login extends React.Component {
...
+                  <span className="fa fa-times-circle" /> Logout
```
Comment (camd) [inline ui/shared/auth/Login.jsx]: The ``fa-sign-out`` icon is more appropriate for this.

**Comment by armenzg** [comment]
Thanks for the PR. Please re-open when you're ready to work on it again and have handled the conflicts.

**Pull request closed without merging**